### ISSUE TITLE
FormWindow response bug

### ISSUE THREAD

**Issue by Creeperface01** [yescallop/Nukkit#67]
### Issue Description
something is wrong with response decoding

### Steps to Reproduce the Issue
tested with custom windows

### Crashdump, Backtrace or Other Files
https://hastebin.com/efirukewuf.css

**Comment by Creeperface01** [comment]
@kvetinac97

**Comment by Goodstuff4UonYT** [comment]
@Creeperface01 @kvetinac97 I agree, if you have two ElementInput, the second one returns null on PlayerFormRespondedEvent.
